spec/unit/builders/ProjectBuilder.spec.js

```diff
@@ -19,7 +19,6 @@
 
 const fs = require('fs');
 const path = require('path');
-const Q = require('q');
```

Comment:
I believe this should remain in this PR and would be removed in : https://github.com/apache/cordova-android/pull/833

Reply:
Module imports are removed _iff_ the module is not used. Otherwise ESLint would complain.